notebooks/official/pipelines/get_started_with_machine_management.ipynb

```diff
@@ -555,19 +505,19 @@
         "\n",
         "Next, set the machine type to use for training and prediction.\n",
         "\n",
-        "- Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you will use for for training and prediction.\n",
+        "- Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for for training and prediction.\n",
```

Comment:
remove one 'for'

Reply:
Instead, it should be: - Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for training and prediction.